src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  font-style: normal;
+
+  --hover-color: #00acdc;
+}
+
+.header {
+  width: 100%;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
```

Comment:
Be careful with setting fixed dimensions to blocks. Let the content size dictate it. To avoid overflow or accidental scroll bar. In your case, you've set a fixed width of 100% to the .header class. If there are no specific design requirements for this, it's better to let the content dictate the width.